doc/direction.md

```diff
@@ -0,0 +1,65 @@
+# Minetest Direction Document
+
+---------------------------
+
+## 0. Preface
+
+### 0.1. Introduction
+
+This document attempts to set out Minetest's philosophy, aims, and objectives.
+
+Please note that Minetest is developed by a community, and not everyone in the community
+holds the same views as to what Minetest should be.
+
+### 0.2. Definitions
+
+* `Minetest` is the game engine.
+* `Minetest Game` is the default game which comes with the engine.
+* The `Minetest Project` refers to the combination of the above, the development team, and supporting tools.
+
+### 0.3. Other documents
+
+celeron55 has published his opinions on the development of Minetest in these places:
+
+* What is Minetest? - http://c55.me/blog/?p=1491
+* Roadmap - https://forum.minetest.net/viewtopic.php?t=9177
+* A clear mission statement for Minetest is missing - https://github.com/minetest/minetest/issues/3476#issuecomment-167399287
+
+Also see the roadmaps and manifestos of core developers on the forums: https://forum.minetest.net/viewforum.php?f=7
+
+---------------------------
+
+## 1. Mission Statement
```

Comment:
I have seen some guys on the forums, who use MT for educational purposes. I think it worth mentioning in mission too.

Reply:
MT For Education isn't a "goal", it's more of a side-effect due to the nature of MT. Facilitating any such "mucking about" is somewhat of the goal of MT, and specific examples of it don't necessarily need to be mentioned.

Reply:
I agree with benrob0329, use in education is not a primary part of the mission.

Reply:
Education is an interesting use case but it isn't necessary to mention it